hdf5.spec

```
@@ -565,6 +569,9 @@ done
 
 
 %changelog
+* Fri Jun 19 2020 Phil Henderson <phillip.henderson@intel.com> - 1.10.5-8.g07066a381e
+- Fix Leap 15 build of %{name}-devel
```

Not necessary in this PR, but an explnation of what a _fix_ was is better than just saying _Fix_.  I.e.

```suggestion
- Leap 15 Requires: gcc-fortran%{?_isa} rather than gcc-gfortran%{?_isa}
```